lua/toggleterm.lua

```diff
@@ -110,11 +110,20 @@ end
 function M.handle_term_enter()
   local _, term = terms.identify()
   if term then
-    term:resize()
```

Comment:
Why remove this? This is there to prevent nvim forcefully resizing the window because of settings like `equalalways` etc.

Reply:
I removed this so that manual resizing while the window is not focused can be remembered. Is there a way to distinguish between automatic resizing and manual resizing?

Reply:
Isn't that already done in `open_split()` even before this PR?

Reply:
@zeertzjq this is here also to catch them case where someone is using vim sessions and the terminal gets "resurrected" vim might not open it to correct size on vim enter

Reply:
I guess it is safe not to call this if `persistent[term.direction] ~= nil` then.

Reply:
Yeah I guess that works since if there is a persistent size already then it is not a newly resurrected terminal.